src/test/java/org/jenkinsci/plugins/electricflow/HtmlSanitizerTest.java

```diff
@@ -0,0 +1,22 @@
+package org.jenkinsci.plugins.electricflow;
+
+import org.jenkinsci.plugins.electricflow.ui.HtmlUtils;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class HtmlSanitizerTest {
+
+    @Test
+    public void simpleSanitize() {
+        String sanitized = HtmlUtils.getHtmlPolicy().sanitize("<html><head><script>alert('paco');</script></head><body><h1>Hi</h1></body></html>");
+        Assert.assertNotNull("Sanitized string should not be null", sanitized);
+        Assert.assertEquals("The sanitized html is not the one expected", "Hi", sanitized);
```

Comment:
confused me for a short while:  `h1` is not allowed -> https://github.com/jenkinsci/electricflow-plugin/blob/df11b959d642ade6a86633741b6b8a1393f8a95f/src/main/java/org/jenkinsci/plugins/electricflow/ui/HtmlUtils.java#L11